index.html

```diff
@@ -12,64 +12,103 @@
 </head>
 
 <body>
-    <div class="container">
```

Comment:
Your header is working fine, good job with this!  You can simplify this a little bit.  You should be able to just use a header/nav instead of wrapping it in the container class, remove the class names on the anchors.  With that being said, there is nothing wrong with this at all, it is preference on how you like the lay things out.